server/services/store/sqlstore/migrations/000034_category_boards_remove_unused_delete_at_column.up.sql

```diff
@@ -0,0 +1,20 @@
+{{ if or .postgres .mysql }}
+    {{ dropColumnIfNeeded "category_boards" "delete_at" }}
+{{end}}
+
+{{if .sqlite}}
```

Comment:
If we want to support dropColumn for Sqlite, then we should add this support to the dropColumnIfNeeded versus adding all this special case in the scripts.

Reply:
@wiggin77 as you can see in the script that I need to create a temp table with the exact same schema as the original table minus the column to be removed. Ther isn't a way to do this in SQLite (correct me if I'm wrong) for an arbitrary tale with arbitrary schema.

Reply:
Ok, let's not drop any columns for Sqlite and keep the migration scripts as simple as possible.  The migration scripts have been by far our biggest source of instability and outages.  We'll be dropping support for Sqlite soon anyway.

Reply:
Makes sense. Done.